app/androidutils.cpp

```diff
@@ -214,6 +226,8 @@ void AndroidUtils::handleActivityResult( int receiverRequestCode, int resultCode
     QString absolutePath = absolutePathJNI.toString();
 
     QString selectedImagePath = "file://" + absolutePath;
+    getExifInfo( absolutePath );
```

Comment:
So we use this all the time for Android? What about setting it on/off in settings or something like that?

Reply:
Good question. There is a space for improvements such as have a setting for it or check if the value exists by any chance. Now, the function is called unconditionally indeed.

Reply:
I would leave it to be ON by default with no setting to turn it OFF for the time being.